mlir/lib/Dialect/Rock/Transforms/AlignTiling.cpp

```diff
@@ -591,31 +618,19 @@ makeExtraInputTile(LinalgAlignRewriter &b, TiledOp tiledOp, Value src,
   LLVM_DEBUG(llvm::dbgs() << "Src type: " << src.getType()
                           << " tile type: " << tile.getType() << "\n");
 
+  // Reset the insertion point if laGeneric and tiledOp are in different blocks.
+  // Insert new transform operations and threadwiseReadIntoOp where
+  // linalg.generic is if linalg.generic reads from the output of tiledOp.
+  // Otherwise, if tiledOp reads from the output of linalg.generic, insert
+  // transformation and threadwiseReadIntoOp where tiledOp is.
```

Comment:
I'm not sure `sink` is a good variable name here.

Also, this comment could use some motivation.